snippets/serverless-kafka-broker-intro.adoc

```diff
@@ -1,7 +1,17 @@
 // Text snippet included in the following modules:
 //
 // * /serverless/develop/serverless-using-brokers.adoc
+// * /serverless/develop/serverless-kafka-developer.adoc
 
 :_content-type: SNIPPET
 
+The Knative Kafka broker is an Apache Kafka native implementation of the Knative broker API that reduces network hops, supports any Kafka version, and has a better integration with Kafka for the broker and trigger model over other broker types.
```

Comment:
Any kafka version: not really sure we wanna say that.

Currently we have some 2.8x and up to 3.0 I think .

I'd just remove the "version"